doc/apiv3-documentation.apib

```diff
@@ -3153,6 +3162,94 @@ Permanently deletes the specified user account.
                 "message": "The specified user does not exist."
             }
 
+# Group UserPreferences
+
+## Linked Properties
```

Comment:
Consistently with other endpoints and according to our HATEOAS principle there should be an action link to `update` the preferences, pointing to the `PATCH` endpoint.